### ISSUE TITLE
Origin not found

### ISSUE THREAD

**Issue by DSIW** [zeapo/Android-Password-Store#492]
Hey,

I'd like to push my repository to the remote one, but I get an error message:

> An error occurred during a git operation
>
> Error occured during the push operation:origin not found.
> Caused by: org.eclipse.jgit.errros.NoRemoteRepositoryException: origin: not found.

Is there a way of pushing my passwords to the remote without loosing my new passwords?

Regarding this https://github.com/zeapo/Android-Password-Store/issues/357#issuecomment-349833720, I don't want to delete my local repository because it contains new passwords which I need.

I also thought of downgrading to v1.2.0... Maybe that helps but I'm not sure.

BTW: Thank you for your great work!!

Cheers,
Max

**Comment by DSIW** [comment]
I tried it again, and now I get:

> An error occurred during a Git operation
>
> Error occurred during the push operation:Attempt to get length of null array
> Caused by:
> null

I used the internal storage and my phone isn't rooted. Can I somehow put the files to another location?

**Comment by DSIW** [comment]
Yes, I found another solution:

TL;DR Export the passwords to your external storage and copy them to your computer.

1. Goto settings > Misc > Export Passwords
1. Select one folder on the external storage (e.g. `/storage/emulated/0/pwd`)
1. Connect your phone with your computer in debug mode
1. Copy the created directory to your computer: `adb pull /storage/emulated/0/pwd/password_store_xxxx_xx_xx_xx_xx_xx/ .`
1. `git push`

Luckily all files including the hidden files are copied, so that git works out of the box. Thank you for that awesome feature! I could have done that earlier but thought it would export it in a unencrypted way.

Now I will reinstall the app and hopefully the git synchronization works then.

**Comment by DSIW** [comment]
Follow up: After recloning the repository, I didn't get the error from above :+1:
